docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -51,16 +66,19 @@
 * I2C EEPROM
 * Bluetooth module via UART
 * IMU (e.g. LSM6DS33)
+* I2C port extender
+  * if port extenders are used for inputs, the interrupt pin has to be connected to an interrupt capable pin of the MCU
 * SpaceMouse module
 
 ---
 
 ## Black-and-white radios
 
 ### Mandatory hardware features (B/W)
-* MCU: STM32F2 and STM32F4 with minimally 512 kByte flash (for new radios only STM32F4 with minimally 1 MB flash)
-  <br/> STM32F2 running at 120 MHz, STM32F4 running at 168 MHz
+* MCU: STM32F4 with minimally 1 MB flash running at 168 MHz
```

Comment:
, whereas from EdgeTX v3 onwards, no new radios with STM32F MCUs will be supported.